spring-pulsar/src/test/java/org/springframework/pulsar/core/PulsarTemplateTests.java

```diff
@@ -236,7 +238,7 @@ void sendMessageWithTopicInferredByTypeMappings(boolean producerFactoryHasDefaul
 	void sendMessageWithoutTopicFails() {
 		PulsarProducerFactory<String> senderFactory = new DefaultPulsarProducerFactory<>(client);
 		PulsarTemplate<String> pulsarTemplate = new PulsarTemplate<>(senderFactory);
-		assertThatIllegalArgumentException().isThrownBy(() -> pulsarTemplate.send("test-message"))
+		assertThatExceptionOfType(PulsarException.class).isThrownBy(() -> pulsarTemplate.send("test-message"))
```

Comment:
With the suggested change, this change is not needed.

Reply:
Indeed and I will revert it